Sources/tuist/tuist.docc/Articles/Tuist/Users/installation.md

```diff
@@ -6,40 +6,40 @@ Learn how to install Tuist in your environment
 
 Tuist is a [command-line application](https://en.wikipedia.org/wiki/Command-line_interface) that you need to install in your environment before you can use it. The installation consists of an executable, dynamic frameworks, and a set of resources (for example, templates). Although you could manually build Tuist from the sources, we recommend using one of the following installation methods to ensure a valid installation.
 
-### Recommended: [rtx](https://github.com/jdx/rtx)
+### Recommended: [mise](https://github.com/jdx/mise)
 
-Tuist defaults to [rtx](https://github.com/jdx/rtx) as a tool to deterministically manage and activate versions of Tuist.
+Tuist defaults to [mise](https://github.com/jdx/mise) as a tool to deterministically manage and activate versions of Tuist.
 If you don't have it installed on your system,
-you can use any of these [installation methods](https://github.com/jdx/rtx#installation).
+you can use any of these [installation methods](https://mise.jdx.dev/getting-started.html).
 Remember to add the suggested line to your shell, which will ensure the right version is activated when you choose a Tuist project directory in your terminal session.
 
-> Note: rtx is recommended over alternatives like [Homebrew](https://brew.sh) because it supports scoping and activating versions to directories, ensuring every environment uses the same version of Tuist deterministically.
+> Note: mise is recommended over alternatives like [Homebrew](https://brew.sh) because it supports scoping and activating versions to directories, ensuring every environment uses the same version of Tuist deterministically.
 
 Once installed, you can install Tuist through any of the following commands:
 
 
 ```bash
 # Tuist
-rtx install tuist@3.36.0      # Install a specific version number
-rtx install tuist@3           # Install a fuzzy version number
-rtx use tuist@3.36.0          # Use tuist-3.36.0 in the current project
-rtx use -g tuist@3.36.0       # Use tuist-3.36.0 as the global default
-rtx install tuist             # Install the current version specified in .tool-versions/.rtx.toml
-rtx use tuist@latest          # Use the latest tuist in the current directory
-rtx use -g tuist@system       # Use the system's tuist as the global default
+mise install tuist@3.36.0      # Install a specific version number
```

Comment:
Users may think about 3.36.0 as the latest version here. I think it would be nice to move `mise install tuist` to the first spot.

Reply:
I moved it to the first stot 👍🏼